app/javascript/components/lease_ups/leaseUpAppPrefModel.js

```diff
@@ -16,8 +16,8 @@ export const buildLeaseUpAppPrefModel = (applicationPreference) => {
     lease_up_status: application.processing_status,
     status_last_updated: application.status_last_updated,
     preference_order: applicationPreference.preference_order,
-    preference_record_type: listingPreference.record_type_for_app_preferences,
-    preference_lottery_rank: applicationPreference.preference_lottery_rank,
+    preference_record_type: applicationPreference.preference_name.includes('General') ? applicationPreference.preference_name : listingPreference.record_type_for_app_preferences,
```

Comment:
We should add a comment here to explain what we're doing since the behavior is weird.

Reply:
pinging on this comment again too

Reply:
Updated